package.json

```diff
@@ -28,10 +28,10 @@
   "devDependencies": {
     "@babel/eslint-parser": "^7.16.3",
     "@folio/eslint-config-stripes": "^5.0.0",
-    "@interactors/with-cypress": "^0.1.3",
+    "@interactors/with-cypress": "^0.1.4",
     "babel-eslint": "^10.1.0",
     "bigtest": "^0.14.0",
-    "cypress": "^9.0.0",
```

Comment:
Everythin works with new versions? Version increments can be not easy and a lot of fixed may be need

Reply:
my tests work. Could you provide your stable tests?

Reply:
Now I don't. I mean if you increment some dependencies, we should get some benefits from new version and we should sure about other functional will work. If you don't sure of it. may be we don't need increment now

Reply:
I see a lot of cypress strange things and hope each new version will get us more convenient and effective tool. My suggestion - if our tests still stable - we can up our dependencies as we wish.  And of cause we have the ability fix version in case of known issues/blockers.

Reply:
In this case I don't see problems because this you increment minor version:
https://docs.cypress.io/guides/references/changelog#9-1-0
But in future I suggest to create separate branch for it and check that all tests works

Reply:
correct, you can see such requirement into draft of code review process -" If you change existing packages versions, run all existing tests to verify that already implemented tests still work" - https://wiki.folio.org/display/DQA/Code+review